### ISSUE TITLE
SET jumping back to Attack Module after attack build

### ISSUE THREAD

**Issue by LEDudester** [trustedsec/social-engineer-toolkit#570]
**Expected Behaviour**

After creating an attack, the web server should start listening and SET should prompt something like "SET web server is now listening.." and wait for access to the freshly built attack.

Attack options: 1, 2, 2, 1, no, 10.0.0.1, 2, 46 (AutoPown), 2, 4444

**Actual Behaviour**

After entering the last option (Port 4444), SET does shortly prompt "SET web server is now listening" but immediately jumps back to the Web Attack Module, again asking for an attack method.

I don't have the impression that SET really builds the attack, it takes no time until SET jumps to the prompt for the attack method. Building AutoPown usually takes a few secs.

Questions:
1) Why does SET not stop and listen after building the attack?
2) How do I verify whether the attack has been built correctly?
All freshly installed and tried to build about 20 times with no success.
Build attacks on other systems many times before, always worked.

**Set Version**

7.7.9 (Blackout) on newest Kali, all updated and upgraded to bleeding edge.
All checks below done and successfully passed as described (kali-rolling et al)

Before opening an issue please make sure you are running the most up to date version of set

We see the same bug reports being opened for issues that have already been fixed in a new release

**Set in Kali**

Please make sure you are running kali-rolling and your system is up to date.
To update your system do the following ``apt update && apt full-upgrade``

Run the following to check your version of Kali ``lsb_release -rdc`` and you should see the following
output
```
Description:	Kali GNU/Linux Rolling
Release:	kali-rolling
Codename:	kali-rolling
````
If you do not see the following output download the ISO from http://cdimage.kali.org/kali-images/kali-weekly/

If you do all of the above and still have an issue then please open a issue.

**Comment by LEDudester** [comment]
After a few very frustrating weeks of error hunting, I just installed Kali Linux 2018.1 and fired up setoolkit.
It works like a charm!
Conclusion: setoolkit in Rel. 2018.3a is buggy and unusable.

**Comment by trustedsec** [comment]
I can confirm this is an issue, temporarily you can edit /etc/setoolkit/set.config to APACHE_SERVER=ON, I have a fix in for version 7.8.

**Comment by LEDudester** [comment]
Hi

I also tried this. Setting Apache to ON did not resolve the issue.

Regards

George
